src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,12 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
+  todos: Todo[],
+};
+
+export const TodoList:React.FC<Props> = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => <TodoInfo todo={todo} />)}
```

Comment:
Don't forget about `key` prop when you render a list. Read more: https://reactjs.org/docs/lists-and-keys.html#extracting-components-with-keys
```suggestion
    {todos.map(todo => <TodoInfo key={todo.id} todo={todo} />)}
```